kratos/modeler/copy_properties_modeler.cpp

```diff
@@ -94,7 +94,7 @@ void CopyPropertiesModeler::RecursivelyCopyProperties(
 template<class TContainerType>
 void CopyPropertiesModeler::ReplaceProperties(
     TContainerType& rContainer,
-    const ModelPart& rModelPart)
+    ModelPart& rModelPart)
```

Comment:
```c++
template<class TContainerType>
void CopyPropertiesModeler::ReplaceProperties(
    TContainerType& rContainer,
    ModelPart& rModelPart)
{
    block_for_each(rContainer, [&](typename TContainerType::value_type& rEntity){
        auto properties_id = rEntity.GetProperties().Id();
        auto p_properties = rModelPart.pGetProperties(properties_id);
//                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
//                                     This method is not const
        rEntity.SetProperties(p_properties);
    });
}
```

It wouldn't compile otherwise. `pGetProperties` is not const (for modelparts) as of this PR.

This is actually an inconsistency within this PR. Modelpart is:
- `Modelpart::pGetProperties -> Properties::PointerToConst` is const
- `Modelpart::pGetProperties -> Properties::Pointer` is not const (This is the one being used here)

Element:
- `Element::pGetProperties -> Properties::Pointer` is const but shouldn't, to avoid breaking `Element::Create`
- `Element::pGetProperties -> Properties::PointerToConst` cannot exist as it would ambiguosly overload the previous one.

Condition:
- `Condition::pGetProperties -> Properties::Pointer` is const but shouldn't, to avoid breaking `Condition::Create`
- `Condition::pGetProperties -> Properties::PointerToConst` cannot exist as it would ambiguosly overload the previous one.


The `Condition` and `Element` cases will be tackled in a future PR as they would break pretty much every single element and condition in Kratos.

The `Modelpart` one is done in this PR. Alternatively we could keep it "wrong" for consistency and fix it the future PR.

I don't mind either way.